src/components/DashboardNoTeam/CreateTeamForm/CreateTeamForm.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Box, Text, Button, Input, Heading, HStack, VStack, FormControl } from '@chakra-ui/react';
+
+const InviteTeamMembers = props => {
+  const handleInputChange = (e, index) => {
+    const { name, value } = e.target;
+    const list = [...props.inputList];
+    list[index][name] = value;
+    props.setInputList(list);
+  };
+
+  const handleRemoveClick = index => {
+    const list = [...props.inputList];
+    list.splice(index, 1);
+    props.setInputList(list);
+  };
+
+  const handleAddClick = () => {
+    props.setInputList([...props.inputList, { email: '' }]);
+  };
+
+  return (
+    <Box w="100%">
+      {props.inputList.map((x, i) => {
+        return (
+          <Box d="flex" mb="8px" key="{x}">
```

Comment:
I don't think you need quotes around {x}. with the quotes, you're giving all the components the same key :p